src/lib/config.ts

```diff
@@ -1670,68 +1699,13 @@ const parseConfigHelper = (pconf, parseobj) => {
     return parseobj
 }
 
-// Listen for changes to the storage and update the USERCONFIG if appropriate.
-// TODO: BUG! Sync and local storage are merged at startup, but not by this thing.
```

Comment:
@arcnmx + @glacambre: Is all of this code I've deleted just to try to work around `storage.onChanged` being a bit pants? Our locks should solve all of these problems. I think. Opinions?